lib/tap_watir/element.rb

```diff
@@ -0,0 +1,35 @@
+require 'watir'
+
+module TapWatir
+  class Element
+
+    def initialize(driver,selector) #breaks PR #7
+      @driver = driver
+      @selector = selector
+    end
+
+    # should be private
+    private
+    def locate
+      @element = @driver.find_element(@selector.keys.first,@selector.values.first)
+    end
+
+    def wd
+      @element || locate
+    end
+
+    def exist?
+
+      return false if @element
+      assert_exists
+    end
+
+
+    def assert_exists? # should I be using rescue ??
```

Comment:
Not sure

Reply:
1. Let's implement this in a separate PR
2. It would get rescued it in `#exists?` method, this method is written correctly
3. Also we should make it private for now